### PR TITLE
Bump main to 5.0.0-dev

### PR DESCRIPTION
**Pull request by ashley-cui**
Dev version bump, plus updated release notes from 4.7

```release-note
None
```

### CODE REVIEW

**Comment by baude** [comment]
LGTM

**Review by Luap99 — APPROVED** [review]
/lgtm
/hold

**Comment by baude** [comment]
just removing the lgtm until we for sure have the versioning figured out

**Comment by mheon** [comment]
I remember us dealing with many of these issues in the 4.0 bump, but aside from the version regex (already fixed) I cannot remember the fixes for the rest.

File: version/version.go [hunk 1]
```diff
@@ -40,7 +40,7 @@ var Version = semver.MustParse(rawversion.RawVersion)
 var APIVersion = map[Tree]map[Level]semver.Version{
 	Libpod: {
 		CurrentAPI: Version,
-		MinimalAPI: semver.MustParse("4.0.0"),
+		MinimalAPI: semver.MustParse("5.0.0"),
```
Comment (mheon) [inline version/version.go:43]: This might need to be kept at 5.0.0-dev until we have 5.0 final

**Comment by mheon** [comment]
I think I can re-add the LGTM given that our API tests are succeeding.
/lgtm

**Comment by baude** [comment]
im guessing we you fix this test that has failed as it will just pass onto the next commit and so -on?

**Comment by mheon** [comment]
I actually don't remember what optional release test does. I do know it only runs on release PRs, so we can merge without worrying about this breaking CI anywhere but release bumps.

**Comment by ashley-cui** [comment]
IIRC The optional release test checks if the version has a -dev suffix or not. It should fail here, so it's not a problem. 

If someone can give this another slash lgtm, this should be ready to go in :)

**Pull request merged**